nemo/collections/asr/models/asr_model.py

```diff
@@ -171,6 +173,51 @@ def on_after_backward(self):
                 logging.warning(f'detected inf or nan values in gradients! Setting gradients to zero.')
                 self.zero_grad()
 
+    @classmethod
```

Comment:
These two methods should be class methods of WithOptionalCYDAGraphs

Reply:
I think, generally, this is not a good idea to introduce a 2-way dependency `WithOptionalCudaGraphs` <-> `ASRModel` (actually, `EncDecRNNTModel`, since decoding is only in this model).
I made the method more abstract to separate the logic, separating the path in the model and the lookup logic.